app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="#">Logo</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Projects</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <% if student_signed_in? %>
+        <%= link_to "Logout",  destroy_student_session_path, :method => :delete   %>
```

Comment:
Create a helper method for it
like.
 <%= link_to "Logout",  signout_path, :method => :delete   %>

in application helper

def signout_path
  student_signed_in? ? destroy_student_session_path : (teacher_signed_in? ? destroy_teacher_session_path : (admin_signed_in? ? destroy_admin_session_path : return ))
end

<% if student_signed_in? %> should be like 
<% if current_user? %>

in application helper
def current_user?
student_signed_in? || teacher_signed_in? || admin_signed_in?
end